pkg/server/server_manifest_test.go

```diff
@@ -308,6 +314,152 @@ func TestManifestAPI(t *testing.T) {
+		// create an issuer
+		issuerDID, err := didService.CreateDIDByMethod(
+			context.Background(),
+			did.CreateDIDRequest{
+				Method:  didsdk.KeyMethod,
+				KeyType: crypto.Ed25519,
+			})
+		assert.NoError(tt, err)
+		assert.NotEmpty(tt, issuerDID)
+
+		// create an applicant
+		applicantDID, err := didService.CreateDIDByMethod(
+			context.Background(),
+			did.CreateDIDRequest{
+				Method:  didsdk.KeyMethod,
+				KeyType: crypto.Ed25519,
+			})
+		assert.NoError(tt, err)
+		assert.NotEmpty(tt, issuerDID)
+
+		// create a schema for the creds to be issued against
+		licenseSchema := map[string]any{
+			"type": "object",
+			"properties": map[string]any{
+				"licenseType": map[string]any{
+					"type": "string",
+				},
+			},
+			"additionalProperties": true,
+		}
+		createdSchema, err := schemaService.CreateSchema(
+			context.Background(),
+			schema.CreateSchemaRequest{Author: issuerDID.DID.ID, Name: "license schema", Schema: licenseSchema, Sign: true})
+		assert.NoError(tt, err)
+		assert.NotEmpty(tt, createdSchema)
+
+		// issue a credential against the schema to the subject, from the issuer
+		createdCred, err := credentialService.CreateCredential(
+			context.Background(),
+			credential.CreateCredentialRequest{
+				Issuer:     issuerDID.DID.ID,
+				Subject:    applicantDID.DID.ID,
+				JSONSchema: createdSchema.ID,
+				Data: map[string]any{
+					"licenseType": "WA-DL-CLASS-A",
+					"firstName":   "Tester",
+					"lastName":    "McTest",
+				},
+			})
+		assert.NoError(tt, err)
+		assert.NotEmpty(tt, createdCred)
+
+		// good request
+		createManifestRequest := getValidManifestRequest(issuerDID.DID.ID, createdSchema.ID)
+
+		requestValue := newRequestValue(tt, createManifestRequest)
+		req := httptest.NewRequest(http.MethodPut, "https://ssi-service.com/v1/manifests", requestValue)
+		w := httptest.NewRecorder()
+		err = manifestRouter.CreateManifest(newRequestContext(), w, req)
+		assert.NoError(tt, err)
+
+		var resp router.CreateManifestResponse
+		err = json.NewDecoder(w.Body).Decode(&resp)
+		assert.NoError(tt, err)
+
+		m := resp.Manifest
+		assert.NotEmpty(tt, m)
+		assert.Equal(tt, m.Issuer.ID, issuerDID.DID.ID)
+
+		// good application request
+		container := []credmodel.Container{{CredentialJWT: createdCred.CredentialJWT}}
+		applicationRequest := getValidApplicationRequest(m.ID, m.PresentationDefinition.ID, m.PresentationDefinition.InputDescriptors[0].ID, container)
+
+		// sign application
+		applicantPrivKeyBytes, err := base58.Decode(applicantDID.PrivateKeyBase58)
+		assert.NoError(tt, err)
+		applicantPrivKey, err := crypto.BytesToPrivKey(applicantPrivKeyBytes, applicantDID.KeyType)
+		assert.NoError(tt, err)
+		signer, err := keyaccess.NewJWKKeyAccess(applicantDID.DID.ID, applicantPrivKey)
+		assert.NoError(tt, err)
+		signed, err := signer.SignJSON(applicationRequest)
+		assert.NoError(tt, err)
+
+		now := time.Date(2022, 10, 31, 0, 0, 0, 0, time.UTC)
```

Comment:
nit: this is not now 😄

Reply:
🤣 

Injected this time into the service now for testing purposes.